src/js/utils/utils.js

```diff
@@ -312,6 +309,54 @@ const utils = {
 
         audio.preload = 'metadata';
         audio.src = path;
+    },
+
+    fetchCover(trackPath, callback) {
+
+        if(!trackPath) {
+            callback(null);
+            return;
+        }
+
+        const stream = fs.createReadStream(trackPath);
+
+        mmd(stream, { duration: true }, (err, data) => {
+
+            if(err) console.warn(err);
```

Comment:
I don't think It's a good idea: 
- the `return` statement should only return a string
- if `musicmetadata` fails, it should still scan the folder for a cover right ?